src/internal/kms.ts

```diff
@@ -67,7 +67,7 @@ export class KMSClient extends AWSClient {
             {}
         )
 
-        const res = http.request(this.method, signedRequest.url, signedRequest.body, {
+        const res = await http.request(this.method, signedRequest.url, signedRequest.body, {
```

Comment:
@oleiade you didn't change this one to `asyncRequest`

Reply:
Good catch, fixing it indeed, thanks 🙇🏻